HerPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -156,7 +156,7 @@ public EpcRating DisplayEpcRating
 
     public bool EpcIsTooHigh => EffectiveEpcBand is EpcRating.A or EpcRating.B or EpcRating.C;
 
-    public bool IncomeIsTooHigh => IncomeBand is (IncomeBandEnum.GreaterThan31000 or IncomeBandEnum.GreaterThan34500) && IsLsoaProperty is not true;
+    public bool IncomeIsTooHigh => IncomeBand is (IncomeBandEnum.GreaterThan31000 or IncomeBandEnum.GreaterThan34500 or IncomeBandEnum.GreaterThan36000) && IsLsoaProperty is not true;

```

Comment:
Think we should clearly mark the fact that we're still using the old bands here (for backwards-compatibility with old sessions). I suppose this is something that marking the values as `Obsolete` would do.